src/content/1.7/development/page-reference/back-office/product/form/_index.md

```diff
@@ -0,0 +1,254 @@
+---
+title: The Product Form
+menuTitle: Product Form
+weight: 42
+---
+
+# The New Product Form
+{{< minver v="1.7.8" title="true" >}}
+
+In most recent Product page, which was remade in 1.7.8, the form is divided into multiple sub forms. Each form has dedicated CQRS commands to manage its content.
+For previous versions you can read this documentation about [product form before 1.7.8]({{< ref "1.7/development/page-reference/back-office/product/form/product-form-before-1-7-8" >}}).
```

Comment:
Consider adding this in a notice tip like here: https://devdocs.prestashop.com/1.7/modules/creation/module-translation/new-system/

Reply:
Done